tests/functional_tests/0097_upgrade_test.py

```diff
@@ -65,11 +66,16 @@ def test_01_upgrade_jail(
 
     jail = jail('upgrade_test')
     # Let's create a jail now of version releases.index(release) - 1
+    if (jail_ip and dhcp) or jail_ip:
```

Comment:
We can just simplify this to just `if jail_ip`, as we don't care if they supplied dhcp or not at that point.